paloalto_wildfire/komand_paloalto_wildfire/actions/submit_url/action.py

```diff
@@ -41,9 +42,34 @@ def run(self, params={}):
         try:
             r = requests.post(url, files=req)
             o = xmltodict.parse(r.content)
+            out = dict(o)
+
+            #self.logger.info(out)
+            #{
+            #   "submission": {
+            #     "error": {
+            #       "error-message": "'Invalid webpage type url, url should start with http or https'"
+            #     }
+            #   }
+            #}
+            if 'submission' in out:
+                if 'error' in out['submission']:
+                    if 'error-message' in out['submission']['error']:
+                      error = out['submission']['error']['error-message']
+                      raise PluginException(cause='Received an error response from Wildfire.', assistance=f'{error}.')
+
+            # A different response occurs sometimes
+            # {'error': OrderedDict([('error-message', "'Invalid webpage type url, url should start with http or https'")])}
+            if 'error' in out:
+                if 'error-message' in out['error']:
+                  error = out['error']['error-message']
+                  raise PluginException(cause='Received an error response from Wildfire.', assistance=f'{error}.')
+                else:
+                  self.logger.info(out)
+                  raise PluginException(cause='Received an error response from Wildfire.', assistance="Check the log output for more details.")
+
             out = dict(o['wildfire']['submit-link-info'])
         except:
```

Comment:
```suggestion
        except as e:
            raise PluginException(cause='Received an error response from Wildfire.', assistance=f"{e}")
```

Reply:
The log output is in `out` which gets printed before the exception is raised. It can be clunky so I opted to not put it in the `data` argument and instead tell the user to look at the output.